.github/workflows/package-and-release.yaml

```diff
@@ -14,6 +14,12 @@ jobs:
   single-card-demos:
     uses: ./.github/workflows/single-card-demo-tests.yaml
     secrets: inherit
+  single-card-device-perf:
+    uses: ./.github/workflows/perf-device-models.yaml
+    secrets: inherit
+  t3000-demos:
+    uses: ./.github/workflows/t3000-demo-tests.yaml
```

Comment:
I'm not sure if this will break if we already build artifacts in a separate workflow?

Reply:
This 100% will and it did. Will do a trick we've been in other workflows and re-update the PR